### PR TITLE
Convert from Travis CI to Github actions

### PR DESCRIPTION
**Pull request by parnic-sks**

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+          - os: openbsd
+            arch: mips64
+          - os: openbsd
+            arch: mips64le
+          - os: openbsd
+            arch: ppc64
+          - os: openbsd
+            arch: ppc64le
+          - os: openbsd
+            arch: s390x
```
Comment (kemenaran) [inline .github/workflows/release.yml:79]: It seems the exclude list is larger than an explicit inclusion list would be. Maybe dropping the matrix and instead specifying each os-arch variant by hand would be simpler? (of course there would be some redundancy, but simple is good and maintainable).

**Review by kemenaran — APPROVED** [review]
Excellent 👏

File: .github/workflows/release.yml (hunk 1 above)
Comment (parnic-sks) [reply .github/workflows/release.yml:79]: I tried a few different ways of making this work, and wasn't able to find another solution that actually ran each os/arch combo without laying them out in the matrix section (https://github.com/parnic-sks/ubiquitous-memory/actions). I don't like this solution either, so if you find a better way, please advise! :)

**Comment by parnic-sks** [comment]
Opening a new PR to drop the Travis requirement (hopefully).

**Pull request closed without merging**